docs/built-in-functions.rst

```diff
@@ -94,25 +93,90 @@ Bitwise Operations
+
+    Deploys a small, EIP1167-compliant, "minimal proxy contract" that duplicates the logic of the contract at ``target``, but has its own state since every call to ``target`` is made using ``DELEGATECALL`` to ``target``. To the end user, this should be indistinguishable from an independently deployed contract with the same code as ``target``.
+
+
+    * ``target``: Address of the contract to proxy to
+    * ``value``: The wei value to send to the new contract address (Optional, default 0)
+    * ``salt``: A ``bytes32`` value utilized by the deterministic ``CREATE2`` opcode (Optional, if not supplied, ``CREATE`` is used)
+
+    Returns the address of the newly created proxy contract. If the create operation fails (for instance, in the case of a ``CREATE2`` collision), execution will revert.
+
+    .. code-block:: python
+
+        @external
+        def foo(_target: address) -> address:
+            return create_minimal_proxy_to(_target)
+
+.. note::
+
+  It is very important that the deployed contract at ``target`` is code you know and trust, and does not implement the ``selfdestruct`` opcode as this will affect the operation of the proxy contract.
+
+.. note::
+
+  There is no runtime check that there is code already deployed at ``target`` (since a proxy may be deployed counterfactually). Most applications may want to insert this check.
 
 .. note::
 
-  It is very important that the deployed contract at ``target`` is code you know and trust, and does not implement the ``selfdestruct`` opcode as this will affect the operation of the forwarder contract.
+  In previous versions of vyper, this function was named ``create_forwarder_to``.
+
+
+.. py:function:: create_copy_of(target: address, value: uint256 = 0[, salt: bytes32]) -> address
 
-    * ``target``: Address of the contract to duplicate
+    Create a physical copy of the runtime code at ``target``. The code at ``target`` is byte-for-byte copied into a newly deployed contract.
+
+    * ``target``: Address of the contract to copy
     * ``value``: The wei value to send to the new contract address (Optional, default 0)
-    * ``salt``: A ``bytes32`` value utilized by the ``CREATE2`` opcode (Optional, if supplied deterministic deployment is done via ``CREATE2``)
+    * ``salt``: A ``bytes32`` value utilized by the deterministic ``CREATE2`` opcode (Optional, if not supplied, ``CREATE`` is used)
 
-    Returns the address of the duplicated contract.
+    Returns the address of the created contract. If the create operation fails (for instance, in the case of a ``CREATE2`` collision), execution will revert. If there is no code at ``target``, execution will revert.
 
     .. code-block:: python
 
         @external
         def foo(_target: address) -> address:
-            return create_forwarder_to(_target)
+            return create_copy_of(_target)
+
+.. py:function:: create_with_code_of(target: address, *args, value: uint256 = 0[, salt: bytes32]) -> address
```

Comment:
hmm, maybe `create_from_factory` might make more sense?

Reply:
i was thinking about this, it makes sense, especially if we can standardize on the "factory" terminology

Reply:
`create_with_factory_at()`?

Reply:
Totally better yeah

Reply:
renamed to `create_from_factory()`